kolibri/plugins/learn/assets/src/vue/side-nav/index.vue

```diff
@@ -74,6 +76,28 @@
     font-size: $font-size
     font-weight: 300
     overflow: hidden
+    position: fixed
+    z-index: 2
+    @media screen and (orientation: landscape)
+      height: 100%
+      top: 0
+      left: 0
+      width: $left-margin - $card-gutter * 0.5
+    @media screen and (orientation: portrait)
+      bottom: 0
+      width: 100%
+      height: auto
+      display: table
```

Comment:
In ca8466db693e9b1b0ee49415123b92ec86938150 I had refactored the nav to avoid using `display: table`, and instead use Jeet's vertical alignment helpers. I'd much prefer to keep it that way. I can make a PR for that if it would be helpful.